user/details/script.js

```diff
@@ -661,6 +664,262 @@ function showContent() {
   section3.classList.remove('hide');
 }
 
+function generateRemainingDays(dateStr) {
+  const date = new Date(dateStr);
+  const now = new Date();
+  const offset = 330 * 60 * 1000;
+  const nowInIST = new Date(now.getTime() + offset);
+  const diff = nowInIST - date;
+  if (diff >= 24 * 60 * 60 * 1000) {
+    return Math.floor(diff / (24 * 60 * 60 * 1000)) + ' days ago';
+  } else if (diff >= 60 * 60 * 1000) {
+    return Math.floor(diff / (60 * 60 * 1000)) + ' hours ago';
+  } else if (diff >= 60 * 1000) {
+    return Math.floor(diff / (60 * 1000)) + ' minutes ago';
+  } else {
+    return Math.floor(diff / 1000) + ' seconds ago';
+  }
+}
+
+function formatDate(dateStr) {
+  const date = new Date(dateStr);
+  const options = {
+    year: 'numeric',
+    month: '2-digit',
+    day: '2-digit',
+    hour: '2-digit',
+    minute: '2-digit',
+    timeZone: 'Asia/Kolkata',
+  };
+  const formattedDate = date.toLocaleDateString('en-US', options);
+  return formattedDate;
+}
+
+function generatePrsTabDetails() {
+  const div = createElement({
+    type: 'div',
+    classList: ['hidden-content', 'hide'],
+  });
+  const prsSec = createElement({
+    type: 'div',
+    classList: ['user-pr'],
+  });
+
+  const pagination = createElement({ type: 'div', classList: ['pagination'] });
+  const prevBtn = createElement({
+    type: 'button',
+    classList: ['pagination-prev-page', 'btn'],
+  });
+  prevBtn.appendChild(createTextNode('Prev'));
+  prevBtn.addEventListener('click', () => {
+    if (currentPageIndex <= 1) return;
+    currentPageIndex--;
+    generateUserPrsList(getPrsToFetch(userAllPrs, currentPageIndex));
+  });
+
+  const nextBtn = createElement({
+    type: 'button',
+    classList: ['pagination-next-page'],
+  });
+  nextBtn.appendChild(createTextNode('Next'));
+  nextBtn.addEventListener('click', () => {
+    if (currentPageIndex >= totalPrsPages) return;
+    currentPageIndex++;
+    generateUserPrsList(getPrsToFetch(userAllPrs, currentPageIndex));
+  });
+  pagination.append(prevBtn, nextBtn);
+
+  div.append(prsSec, pagination);
+  document.querySelector('.accordion-prs').appendChild(div);
```

Comment:
OPINION: `document.querySelector('.accordion-prs')` can be extracted into a variable and declared on top of the function for better readability.

Reply:
done